contracts/teller-network/Fees.sol

```diff
@@ -51,26 +52,47 @@ contract Fees is Ownable {
     }
 
     /**
-     * @notice Withdraw fees by sending them to the fee destination address
-     */
-    function withdrawFees(address _tokenAddress) public {
-        uint fees = feeTokenBalances[_tokenAddress];
-        feeTokenBalances[_tokenAddress] = 0;
-        if (_tokenAddress == address(0)) {
-            require(address(this).balance >= fees, "Not enough balance");
-            feeDestination.transfer(fees);
-            emit MyEvent(fees, feeDestination, address(this).balance);
+     * @notice Release fee to fee destination and arbitrator
+     * @param _arbitrator Arbitrator address to transfer fee to
+     * @param _value Value sold in the escrow
+     * @param _isDispute Boolean telling if it was from a dispute. With a dispute, the arbitrator gets more
+    */
+    function releaseFee(address payable _arbitrator, uint _value, address _tokenAddress, bool _isDispute) internal {
+        uint _milliPercentToArbitrator;
+        if (_isDispute) {
+            _milliPercentToArbitrator = 100000; // 100%
+        } else {
+            _milliPercentToArbitrator = 10000; // 10%
+        }
+
+        uint feeAmount = getValueOffMillipercent(_value, feeMilliPercent);
+        uint arbitratorValue = getValueOffMillipercent(feeAmount, _milliPercentToArbitrator);
+        uint destinationValue = feeAmount - arbitratorValue;
+
+        if (_tokenAddress != address(0)) {
+            ERC20Token tokenToPay = ERC20Token(_tokenAddress);
+            // Arbitrator transfer
+
+            uint allow = tokenToPay.allowance(address(this), address(this));
```

Comment:
`allow` and `myEvent` are probably part of a debugging session and should be removed